addons/mail/models/mail_alias.py

```diff
@@ -85,8 +85,9 @@ def _alias_is_ascii(self):
             local-part. Quoted-string and internationnal characters are
             to be rejected. See rfc5322 sections 3.4.1 and 3.2.3
         """
-        if self.alias_name and not dot_atom_text.match(self.alias_name):
-            raise ValidationError(_("You cannot use anything else than unaccented latin characters in the alias address."))
+        for alias in self:
+            if alias.alias_name and not dot_atom_text.match(alias.alias_name):
```

Comment:
if any(alias.alias_name and not dot_atom_text.match(alias.alias_name) for alias in self) ?

Reply:
@tde-banana-odoo  This was not needed for FIX.

However, I have Improved this.